recipes/ipywe/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set gitorg = "scikit-beam" %}
+{% set name = "ipywe" %}
+{% set version = "condaforge" %}
+{% set sha256 = "464c20e8e10dd68f1cd7f4c96b2dc2014733b47e37de9340ddd23238ee14818c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+#  url: https://github.com/{{ gitorg }}/{{ name }}/archive/{{ version }}.tar.gz
+  url: https://github.com/{{ gitorg }}/{{ name }}/archive/conda-forge.tar.gz
```

Comment:
Please point it to a release, like https://github.com/scikit-beam/ipywe/archive/0.1.1.tar.gz or https://pypi.org/project/ipywe/0.1.1a1/